components/search/testimony/useTestimonyRefinements.tsx

```diff
@@ -1,41 +1,34 @@
-import { useRefinementListUiProps } from "@alexjball/react-instantsearch-hooks-web"
 import { useRefinements } from "../useRefinements"
-import { useCallback } from "react"
-import { RefinementListItem } from "instantsearch.js/es/connectors/refinement-list/connectRefinementList"
 
 export const useTestimonyRefinements = () => {
   const baseProps = { limit: 500, searchable: true }
   const propsList = [
-    useRefinementListUiProps({
-      transformItems: useCallback(
```

Comment:
@kiminkim724 Want to get your take on this - does this look like a problem for the Private profile changes Boyu started? Ideally, we shouldn't be relying on client-side filtering to hide private data anyway (since the sensitive data would presumably still be accessible by looking at the typesense API results in the network tab and removing results like this can mess with pagination).

Reply:
We don't necessarily need to fix that in this PR - just want to call it out since we need an answer before we can launch the private profile feature.

Reply:
Fixed! I think rebase shenanigans messed up my branch, but I've restored the transform for testimony filtering